spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -1034,14 +1040,22 @@
 
     context "pediatric neurology brief collection" do
       before do
-        make_collection(user, title: "Pediatric Neurology Briefs: Volume 05, Issue 07", id: "pnb-5-7", member_ids: [generic_file_pnb.id])
-
         collection_store.build_collection_store_data
         collection_store.build_paths_for_collection_store
       end
 
-      it "returns the correct community string" do
-        expect(pnb_collection.send(:dh_collection_to_prism_community_collection)).to eq(expected_pnb_community_string)
+      describe "it returns the correct community string" do
+        context "for a pnb generic file" do
```

Comment:
Oh , this needs to change.

Reply:
I misunderstood some stuff in the specs and put down incorrect information in the context strings, but this is corrected now. The actual specs were fine, just the wrong objects being referred to in the context.